example-library/pom.xml

```diff
@@ -27,11 +27,29 @@
     <artifactId>example-library</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>${protobuf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.netflix.eureka</groupId>
```

Comment:
I wouldn't have thought the example library would need the eureka stuff, at least not at runtime. I think we can mark these dependancies as provided, given the services should already have them

Reply:
This is still needed to add the resources and serialiser to the respective caches at the moment. Once these have been addressed then this dependency can be removed from the example-library

Reply:
This is still required by the example configurator to add the serialiser and resources to the relevant caches. Once the resource-service has been updated this can probably be removed.

Reply:
This should be in eample-model then

Reply:
It is already in example-model. Removed from pom.